greenhouse/web/static/style.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-FileCopyrightText: 2023 peepo.world developers
+ *
+ * SPDX-License-Identifier: EUPL-1.2
+ */
+
+@import "node_modules/bulma/sass/utilities/initial-variables";
+
+
+
+@import "node_modules/bulma/bulma";
```

Comment:
Nothing for you to do here, but I just wanted to note that I plan to change this to use git submodules node, like in https://github.com/FFY00/ffy00.github.io/.